api/base/exceptions.py

```diff
@@ -1,5 +1,36 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    # Import inside method to avoid errors when the OSF is loaded without Django
+    from rest_framework.views import exception_handler
+    response = exception_handler(exc, context)
+
+    # Error objects may have the following members. Title removed to avoid clash with node "title" errors.
```

Comment:
I see. We might want to handle this by not using the default `exception_handler`. As long as we aren't using the `title` key in our error objects, this is fine for now.